src/Build/Graph/ProjectGraph.cs

```diff
@@ -43,16 +57,41 @@ public ProjectGraph(string entryProjectFile)
         /// <param name="entryProjectFile">The project file to use as the entry point in constructing the graph</param>
         /// <param name="projectCollection">The collection with which all projects in the graph should be associated. May not be null.</param>
         /// <param name="globalProperties">The global properties to use for all projects. May be null, in which case the containing project collection's global properties will be used.</param>
-        /// <param name="toolsVersion">The tools version. May be null.</param>
+        /// <param name="toolsVersion">The tools version to use for all projects in the graph. May be null, in which case the project collection's default tools version will be used.</param>
         /// <exception cref="InvalidProjectFileException">If the evaluation of any project in the graph fails.</exception>
         public ProjectGraph(
             string entryProjectFile,
             ProjectCollection projectCollection,
             Dictionary<string, string> globalProperties,
             string toolsVersion)
```

Comment:
I really think toolsVersion should be hidden from the API entirely. **When** we deprecate it, it just means we have to change the api here as well.